content/custompages/community.html

```diff
@@ -35,12 +35,23 @@ <h2  id="mailinglist">Mailing List</h2>
 		<p>If you intend to extend or contribute to the OpenCMISS codebase, you can subscribe to the OpenCMISS Developers mailing list using the above link. Once subscribed you will receive future developer notifications and exchanges and be able to take part by mailing the developer list at above address.</p>
 	</div>
 </div>
-<h2  id="bugtracker">Bug tracker</h2>
-<p> The bug tracker is where we track bugs, new features, design ideas and occasionally user queries. We only ever add code changes for items in the tracker.</p>
-<p class="accent"><a href="https://tracker.physiomeproject.org/buglist.cgi?query_format=advanced&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&product=Zinc%20library">OpenCMISS-Zinc Tracker (including PyZinc)</a></p>
-<p class="accent"><a href="https://tracker.physiomeproject.org/buglist.cgi?query_format=advanced&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&product=OpenCMISS-Iron">OpenCMISS-Iron Tracker (including PyIron)</a></p>
+<h2  id="GitHubIssues">GitHub issues</h2>
+<p> We track bugs, new features, design ideas and user queries through GitHub issues. We only ever add code changes for items with GitHub issues.</p>
+<p class="accent"><a href="https://github.com/OpenCMISS/zinc/issues">OpenCMISS-Zinc issues (including PyZinc)</a></p>
+<p class="accent"><a href="https://github.com/OpenCMISS/iron/issues">OpenCMISS-Iron issues (including PyIron)</a></p>
+<p class="accent"><a href="https://github.com/OpenCMISS/documentation/issues">OpenCMISS-Documentation issues</a></p>
+<p class="accent"><a href="https://github.com/OpenCMISS/manage/issues">OpenCMISS-Manage comprehensive build system issues</a></p>
+<p class="accent"><a href="https://github.com/OpenCMISS/manage/issues">OpenCMISS-Setup simplified build system issues</a></p>
```

Comment:
Also I think this should be something like:

OpenCMISS-Setup simplified OpenCMISS installation script issues